### ISSUE TITLE
make: recipe for target 'unit-test' failed

### ISSUE THREAD

**Issue by cznic** [oniony/TMSU#147]
While trying to install TMSU as a https://github.com/c-testsuite/c-testsuite dependency (cc @andrewchambers), I'm observing

```
==== jnml@4670:~/src/github.com/oniony/TMSU> go version
go version go1.11 linux/amd64
==== jnml@4670:~/src/github.com/oniony/TMSU> go clean -cache ; rm -rf ~/pkg ; make

CLEANING

go clean github.com/oniony/TMSU
rm -Rf bin
rm -Rf tmsu-x86_64-0.7.0
rm -f tmsu-x86_64-0.7.0.tgz

COMPILING

go build -o bin/tmsu github.com/oniony/TMSU

PACKAGING DISTRIBUTABLE

cp -R bin -t tmsu-x86_64-0.7.0
cp README.md -t tmsu-x86_64-0.7.0
cp COPYING.md -t tmsu-x86_64-0.7.0
cp misc/bin/* -t tmsu-x86_64-0.7.0/bin/
gzip -fc misc/man/tmsu.1 >tmsu-x86_64-0.7.0/man/tmsu.1.gz
cp misc/zsh/_tmsu -t tmsu-x86_64-0.7.0/misc/zsh/
tar czf tmsu-x86_64-0.7.0.tgz tmsu-x86_64-0.7.0

RUNNING UNIT TESTS

go test github.com/oniony/TMSU/...
# github.com/oniony/TMSU/vfs
src/github.com/oniony/TMSU/vfs/fusevfs.go:496: Fatal call has possible formatting directive %v
?   	github.com/oniony/TMSU	[no test files]
ok  	github.com/oniony/TMSU/cli	0.025s
ok  	github.com/oniony/TMSU/common	0.040s
?   	github.com/oniony/TMSU/common/filesystem	[no test files]
ok  	github.com/oniony/TMSU/common/fingerprint	0.130s
?   	github.com/oniony/TMSU/common/log	[no test files]
ok  	github.com/oniony/TMSU/common/path	0.001s
?   	github.com/oniony/TMSU/common/terminal	[no test files]
?   	github.com/oniony/TMSU/common/terminal/ansi	[no test files]
ok  	github.com/oniony/TMSU/common/text	0.008s
ok  	github.com/oniony/TMSU/entities	0.015s
ok  	github.com/oniony/TMSU/query	0.004s
?   	github.com/oniony/TMSU/storage	[no test files]
?   	github.com/oniony/TMSU/storage/database	[no test files]
?   	github.com/oniony/TMSU/version	[no test files]
ok  	github.com/oniony/TMSU/src/github.com/oniony/TMSU/cli	0.015s
ok  	github.com/oniony/TMSU/src/github.com/oniony/TMSU/common	0.025s
ok  	github.com/oniony/TMSU/src/github.com/oniony/TMSU/common/fingerprint	0.129s
ok  	github.com/oniony/TMSU/src/github.com/oniony/TMSU/common/path	0.002s
ok  	github.com/oniony/TMSU/src/github.com/oniony/TMSU/common/text	0.004s
ok  	github.com/oniony/TMSU/src/github.com/oniony/TMSU/entities	0.001s
ok  	github.com/oniony/TMSU/src/github.com/oniony/TMSU/query	0.017s
# github.com/oniony/TMSU/src/github.com/oniony/TMSU/vfs
src/github.com/oniony/TMSU/vfs/fusevfs.go:496: Fatal call has possible formatting directive %v
Makefile:39: recipe for target 'unit-test' failed
make: *** [unit-test] Error 2
==== jnml@4670:~/src/github.com/oniony/TMSU> 
```

**Comment by oniony** [comment]
I think this is because I'm calling log.Fatal rather than log.Fatalf. The compiler has obviously got smarter at detecting this type of programmatic error.

**Comment by oniony** [comment]
Fixed in 2e646b7.
